sdk/python/kfp/gcp.py

```diff
@@ -85,3 +87,17 @@ def _set_tpu_spec(task):
         return task
 
     return _set_tpu_spec
+
+def use_preemptible_vm_gpu(toleration: V1Toleration = V1Toleration(effect='NoSchedule',
+                                                             key='preemptible',
+                                                             operator='Equal',
+                                                             value='true')):
+  """An operator that configures the GKE preemptible in a container op.
+  """
+
+  def _set_preemptible(task):
+    task.add_toleration(toleration)
```

Comment:
Do you need both?

Reply:
Toleration is to allow the pod to be scheduled to the preemptible VM; node selector is lock the pod to be scheduled to the preemptible VM.